desktop/cmp/treemap/TreeMapModel.js

```diff
@@ -324,28 +331,14 @@ export class TreeMapModel extends HoistModel {
         const heatValues = [];
         this.store.records.forEach(it => {
             const val = it.get(heatField);
-            if (this.valueIsValid(val)) heatValues.push(val);
+            if (this.valueIsValid(val)) heatValues.push(Math.abs(val));
         });
 
-        // 2) Split heat values into positive and negative
-        let [posHeatValues, negHeatValues] = partition(heatValues, it => it > 0);
-        posHeatValues = sortBy(posHeatValues);
-        negHeatValues = sortBy(negHeatValues.map(Math.abs));
-
-        // 3) Calculate bounds and midpoints for each range
-        let minPosHeat = 0, midPosHeat = 0, maxPosHeat = 0, minNegHeat = 0, midNegHeat = 0, maxNegHeat = 0;
-        if (posHeatValues.length) {
-            minPosHeat = posHeatValues[0];
-            midPosHeat = posHeatValues[Math.floor(posHeatValues.length / 2)];
```

Comment:
So I am still just now trying to Grok the Balanced mode.  It seems like it distorts the data to try to spread it out a bit more across the range.  But in any case, it seems like the use of the Median, and the "min" that we are using are now changed (min now zero)

Would like to discuss with ATM, but I think since it is not the default, we could potentially drop it, if it no longer made sense to us.

Reply:
Yeah, the intention is balanced mode is to make a nicer looking chart where you have outliers. It does not do an 'accurate' mapping of color to value like 'linear' does, but grants a wider proportion of the range to whichever half is more heavily weighted so you can more easily see the difference.

It's a bit of a philosophical question whether its better to have accurate colors or instead display colors that are easier to compare against each other. The move towards having a set maxHeat certainly seems like we're favoring accuracy over aesthetics, so would be happy to drop it.